From 590fc0651e70a441d060ae4d292d582fd609ece2 Mon Sep 17 00:00:00 2001
From: De Huo <De.Huo@windriver.com>
Date: Fri, 9 Aug 2019 17:35:51 +0800
Subject: [PATCH] Use rule audit_rules_file_deletion_events to replace
 audit_rules_file_deletion_events_*

Use rule audit_rules_file_deletion_events to replace
 audit_rules_file_deletion_events_* rules in WRLinux1019 profile.

Upstream-Status: Pending

Signed-off-by: De Huo <De.Huo@windriver.com>
---
 wrlinux1019/profiles/draft_stig_wrlinux_disa.profile | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile b/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
index 90c6cf0..9872495 100644
--- a/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
+++ b/wrlinux1019/profiles/draft_stig_wrlinux_disa.profile
@@ -173,11 +173,7 @@ selections:
     - audit_rules_unsuccessful_file_modification_open_by_handle_at
     - audit_rules_unsuccessful_file_modification_truncate
     - audit_rules_unsuccessful_file_modification_ftruncate
-    - audit_rules_file_deletion_events_rename
-    - audit_rules_file_deletion_events_renameat
-    - audit_rules_file_deletion_events_rmdir
-    - audit_rules_file_deletion_events_unlink
-    - audit_rules_file_deletion_events_unlinkat
+    - audit_rules_file_deletion_events
     - audit_rules_kernel_module_loading_init
     - audit_rules_kernel_module_loading_delete
     - audit_rules_kernel_module_loading_insmod
-- 
1.9.1

